backend/src/main/java/com/znane_groby/backend/model/Graveyard.java

```diff
@@ -0,0 +1,37 @@
+package com.znane_groby.backend.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import lombok.*;
+import org.springframework.hateoas.RepresentationModel;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
```

Comment:
To wszystko robi `@Data`, dużo wygodniej. Doda też hashcode, to string i equals.

Reply:
ok